pages/api/connect.ts

```diff
@@ -0,0 +1,23 @@
+import { Exm } from '@execution-machine/sdk';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  try {
+    const functionID = `${process.env.FUNCTION_ID}`,
+      exm = new Exm({
```

Comment:
MMMM. Don't create an instance in every request, it's an overkill. Create a single instance outside of the function and use it inside